02-intermediate-ruby/testing-with-minitest.md

```diff
@@ -259,7 +259,15 @@ You may remember from the [Introduction to Automated Testing](../00-programming-
 1. The seconds are greater than or equal to 60, should raise an error
 1. Any parameter is less than 0, should raise an error
 
-**Question** Are there other edge-cases we should test for?  Do any other inputs cause different behaviors in the method?  Talk about it with your SeatSquad members.  If you are unsure check [here](https://gist.github.com/ada-instructor-1/27c2cb78b177a6139241860a6bc54712).
+<details>
+<summary>
+<b>Question:</b> Are there other edge-cases we should test for?  Do any other inputs cause different behaviors in the method?  Talk about it with your SeatSquad members, and click here for an answer if you are still unsure.
```

Comment:
I thought we'd agreed to remove `SeatSquad` from the textbook curriculum.  I actually don't mind it, just looking for consistency.

Reply:
I think that's true, but this PR is not the place for that work.